packages/jira-adapter/test/weak_references/automation_projects.test.ts

```diff
@@ -22,6 +22,10 @@ describe('automation_projects', () => {
   let projectInstance: InstanceElement
   let instance: InstanceElement
   let elementsSource: ReadOnlyElementsSource
+  const AdapterConfigType = new ObjectType({
+    elemID: new ElemID('adapter', 'AdapterConfig'),
+  })
+  const adapterConfig = new InstanceElement('settings', AdapterConfigType)
```

Comment:
adapter config is a singleton, the instance name should not be `settings`
Ideally this should use the real config type, but if it not possible, lets at least make it similar to a real configuration

```suggestion
  const AdapterConfigType = new ObjectType({
    elemID: new ElemID('adapter'),
    isSettings: true
  })
  const adapterConfig = new InstanceElement(ElemID.CONFIG_NAME, AdapterConfigType)
```